interpreter/cling/lib/Interpreter/IncrementalExecutor.h

```diff
@@ -171,8 +172,10 @@ namespace cling {
     }
 
     ///\brief Unload a set of JIT symbols.
-    bool unloadFromJIT(llvm::Module* M, Transaction::ExeUnloadHandle H) {
-      auto iMod = std::find(m_ModulesToJIT.begin(), m_ModulesToJIT.end(), M);
+    bool unloadFromJIT(std::shared_ptr<llvm::Module> M,
```

Comment:
Here, ownership might make sense (during the lifetime of the unload operation). But then this should take a `const shared_ptr&`.

Reply:
Done.